frontend/src/metabase/meta/metadata/Field.js

```diff
@@ -35,7 +36,7 @@ export default class Field extends Base {
     isCategory()  { return isCategory(this._object); }
     isMetric()    { return isMetric(this._object); }
     isDimension() { return isDimension(this._object); }
-    isID()        { return this.special_type === "id"; }
+    isID()        { return isPK(this.special_type); }
```

Comment:
Is there a reason isPK isn't in this Field class vs being a standalone function? @tlrobinson I'd love to try to standardize on one of these two approaches.